Integrations/integration-TruSTAR.yml

```diff
@@ -147,6 +182,172 @@ script:
         d = datetime.datetime.strptime(timestamp, "%Y-%m-%d %H:%M:%S")
         return int(d.strftime("%s")) * 1000
 
+    def create_file_ec(indicators, file, threshold):
+        if not indicators:
+            return {
+                'DBotScore': {
+                    'Indicator': file,
+                    'Type': 'file',
+                    'Score': 0,
+                    'Vendor': 'TruSTAR',
+                }
+            }
+        trustar_ec = {}
+        file_ec = {}
+        dbot_ec = {}
+        for indicator in indicators:
+            file_ec.update({
+                indicator['indicatorType']: indicator['value'],
+            })
+            trustar_ec.update({
+                'Value': indicator['value'],
+                'Whitelisted': indicator['whitelisted'],
+                'Priority': indicator['priorityLevel']
+            })
+            indicator_score = priority_level_to_score(indicator['priorityLevel'])
+            dbot_ec.update({
+                'Indicator': file,
+                'Type': 'file',
+                'Vendor': 'TruSTAR',
+                'Score': 0 if indicator_score == 0 else (2 if threshold > indicator_score else 3)
+            })
+            if threshold <= indicator_score:
+                file_ec.update({
+                    'Malicious': {
+                        'Vendor': 'TruSTAR'
+                    }
+                })
+        return {
+            outputPaths['dbotscore']: dbot_ec,
+            outputPaths['file']: file_ec,
+            'TruSTAR.File': trustar_ec
+        }
+
+    def create_ip_ec(indicators, ip, threshold):
+        if not indicators:
+            return {
+                'DBotScore': {
+                    'Indicator': ip,
+                    'Type': 'ip',
+                    'Score': 0,
+                    'Vendor': 'TruSTAR',
+                }
+            }
+        trustar_ec = {}
+        ip_ec = {}
+        dbot_ec = {}
+        for indicator in indicators:
+            ip_ec.update({
+                'Address': indicator['value'],
+            })
+            trustar_ec.update({
+                'Value': indicator['value'],
+                'Whitelisted': indicator['whitelisted'],
+                'Priority': indicator['priorityLevel']
+            })
+            indicator_score = priority_level_to_score(indicator['priorityLevel'])
+            dbot_ec.update({
+                'Indicator': ip,
+                'Type': 'ip',
+                'Vendor': 'TruSTAR',
+                'Score': 0 if indicator_score == 0 else (2 if threshold > indicator_score else 3)
+            })
+            if threshold <= indicator_score:
+                ip_ec.update({
+                    'Malicious': {
+                        'Vendor': 'TruSTAR',
+                        'Description': indicator['priorityLevel']
```

Comment:
Please provide better Description, such as `Priority level above HIGH`

Reply:
@mayagoldb is that ok?